README.md

```diff
@@ -0,0 +1,69 @@
+
+## Acknowledgements
+
+ - I would like to express my gratitude to the Meta training academy, which teaches all programming skills.
+
+
+## Appendix
+
+- I present a bookstore to your attention. This project was inspired by the fact that 
+now few people read books and I wish it was accessible and simple.
+
+
+## Authors
+
+- Oleksandr Radovets
+
+
+## Badges
+
+- Copyright (c) [2023] [BookStore]
+
+Permission is hereby granted, free of charge, to any person obtaining a copy
+of this software and associated documentation files (the "Software"), to deal
+in the Software without restriction, including without limitation the rights
+to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+copies of the Software, and to permit persons to whom the Software is
+furnished to do so, subject to the following conditions:
+
+The above copyright notice and this permission notice shall be included in all
+copies or substantial portions of the Software.
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+SOFTWARE.
+
+
+## Demo
+
+https://media.giphy.com/media/fsXOS3oBboiYf6fSsY/giphy.gif
```

Comment:
better not to confuse others with such mem-gif in the Demo section. better replace with the real demo video or remove it at all
```suggestion
```